generated/proto/aggregationpb/aggregation.proto

```diff
@@ -65,3 +49,9 @@ enum AggregationType {
   P999 = 21;
   P9999 = 22;
 }
+
+// AggregationID is a unique identifier uniquely identifying
+// one or more aggregation types.
+message AggregationID {
+  uint64 id = 1;
```

Comment:
How do you plan to upgrade when we have more than 64 types?

Reply:
I'd like to revisit if this is useful at all and may remove this type if not, but if we do use it, we can add a new field (e.g., `ids` with a `repeated uint64` type) to accommodate.

Reply:
could possibly just add another uint64 field when the length of ID increased as well.

Reply:
Sure, although that means you'll keep adding uint64 fields as the list of supported types grows. Might be better to just add a `repeated` field in that case.